### ISSUE TITLE
Downloaded SVG file from exporting.buttons.contextButton.menuItems is corrupt when supporting accessibility

### ISSUE THREAD

**Issue by tongyu-microsoft** [highcharts/highcharts#17390]
#### Expected behaviour
The downloaded VG image should be rendered correctly.

#### Actual behaviour
Hit rendering errror when open the downloaded SVG file:
![image](https://user-images.githubusercontent.com/91754176/173622299-50cbe340-6e68-4594-9e23-2bdafa7015ba.png)

#### Live demo with steps to reproduce
1. Add the highcharts accessibility module to the imported Highcharts namespace by `Accessibility.default(Highcharts);`
2. Downloaded a file with SVG type from the menuItems in the plot.
![image](https://user-images.githubusercontent.com/91754176/173622778-ba29682c-0078-476f-aab1-b1b416ffc259.png)
3. Open the SVG file with browser (any browser), hitting rendering error: 
![image](https://user-images.githubusercontent.com/91754176/173622299-50cbe340-6e68-4594-9e23-2bdafa7015ba.png)
4. Open the SVG file with Visual Studio Code, found the issue is that, accessibility module adds a div element to the svg file that we are trying to download (see the red underline in screenshot below).:
![image](https://user-images.githubusercontent.com/91754176/173623127-6bfd4e55-568d-427b-b6c0-1e9a84b4859d.png)
5. If removes the div above, the SVG file can be rendered successfully. The div here shouldn't be downloaded together with the svg element.

#### Product version
Highcharts 9.3.0

#### Affected browser(s)
All browsers

**Comment by pawellysy** [comment]
Hi @tongyu-microsoft, thanks for creating this ticket. Indeed this was an issue in the v9.3.0, but it was already fixed here: https://github.com/highcharts/highcharts/issues/16589

For a fix, please upgrade to the newest version, and for a workaround, see the issue above.

Closing as a duplicate
